addons/website_livechat/models/website.py

```diff
@@ -15,4 +15,4 @@ class WebsiteConfigSettings(models.TransientModel):
 
     _inherit = 'website.config.settings'
 
-    channel_id = fields.Many2one('im_livechat.channel', string='Website Live Chat Channel', related='website_id.channel_id')
+    channel_id = fields.Many2one('im_livechat.channel', string='Channel', related='website_id.channel_id')
```

Comment:
Same as other occurences: the simplified string makes the field difficult to understand outside of the file scope. Better to change labels I think.